Select by Tag Name/app.js

```diff
@@ -0,0 +1,7 @@
+// Complete the challenge by writing JavaScript below
+const colors = ["#C2272D", "#F8931F", "#FFFF01", "#009245", "#0193D9", "#0C04ED", "#612F90"];
+let listItems = document.getElementsByTagName('li');
```

Comment:
Please use `const` since the value was not reassigned.

Reply:
Thanks, @lycoangelo, This is now updated.